UnitsNet/UnitDefinitions/MassFlow.json

```diff
@@ -32,6 +32,18 @@
         }
       ]
     },
+	{
+      "SingularName": "PoundPerHour",
+      "PluralName": "PoundsPerHour",
+      "FromUnitToBaseFunc": "x/0.125998",
+      "FromBaseToUnitFunc": "x*0.125998",
```

Comment:
As commented in test, I think you want to switch these two around.

![image](https://cloud.githubusercontent.com/assets/787816/23874045/8df76e6e-0834-11e7-98e2-6e8ffa0d5316.png)